int32 strategy
int32 set_finish
int32[] init_pos
---
bool goap_return
